src/index.d.ts

```diff
@@ -0,0 +1,108 @@
+import { Point, Polygon, Feature, BBox } from "geojson";
+import { Georaster } from "georaster";
+
+// re-rexport for easy access by downstream user
+export { Georaster } from "georaster"
+
+type GeoblazeBBox = {
+  xmin: number;
+  xmax: number;
+  ymin: number;
+  ymax: number;
+};
+
+interface HistogramOptions {
+  scaleType: "nominal" | "ratio";
+  /** required for ratio scaleType */
+  numClasses?: number;
+  /** required for ratio scaleType */
+  classType?: "equal-interval" | "quantile";
+}
+
+interface Histogram {
+  [binKey: string]: number;
+}
+
+// Some geoblaze methods accept a geometry, and its accepted in a variety of
+// forms.  These types group them for easy reuse
+type InputPolygon = number[][][] | Polygon | Feature<Polygon>;
+type InputPoint = number[] | Point | Feature<Point>;
```

Comment:
Looks like FeatureCollection is also supported according to https://github.com/GeoTIFF/geoblaze/blob/master/src/utils/utils.module.js#L236.  Not documented clearly.